DocumentsFromSnapshotMigration/src/test/java/org/opensearch/migrations/bulkload/EndToEndTest.java

```diff
@@ -139,6 +154,11 @@ private void migrationDocumentsWithClusters(
                     "[{\"NoopTransformerProvider\":{}}]" // skip transformations including doc type removal
                     : null;
 
+            // Add --allow-loose-version-matching flag for loose compatibility
+            if (VersionMatchers.isES_7_0_to_7_9.test(sourceVersion)) {
+                testDocMigrationContext.allowLooseVersionMatches = true;
+            }
```

Comment:
This parameters forces customers to do a parameter update/redeploy and we know for both of these versions it will work.  Since we are now testing this versions and know they work lets not require the flag for these cases.